bin/check-fs-writable.rb

```diff
@@ -143,6 +143,9 @@ def manual_test
       file.close
       file.unlink
     end
+  rescue Errno::EPERM, Errno::ENOENT, Errno::EIO, Errno::EACCES, Errno::ENOSPC, Errno::EROFS => e
+  $stderr.puts "Caught the exception: #{e}"
+  exit -1
```

Comment:
I dont think we should exit with `-1` if anything it should be a 1 but anyways I also feel like this should be either an `unknown`, `warning`, or `critical` which also would remove needing to specifically redirect the output to stderr.